analysis/table2/table2.R

```diff
@@ -0,0 +1,254 @@
+# Load libraries ---------------------------------------------------------------
+print('Load libraries')
+
+library(readr)
+library(dplyr)
+library(magrittr)
+
+# Define table1 output folder ---------------------------------------------------------
+print("Creating output/table2 output folder")
+
+table2_dir <- "output/table2/"
+fs::dir_create(here::here(table2_dir))
+
+# Specify redaction threshold --------------------------------------------------
+print('Specify redaction threshold')
+
+threshold <- 6
+
+# Source common functions ------------------------------------------------------
+print('Source common functions')
+
+source("analysis/utility.R")
+
+# Specify arguments ------------------------------------------------------------
+print('Specify arguments')
+
+args <- commandArgs(trailingOnly = TRUE)
+
+if (length(args) == 0) {
+  cohort <- "prevax"
+  focus <- "history"
+} else {
+  cohort <- args[[1]]
+  focus <- args[[2]]
+}
+
+# Load active analyses ---------------------------------------------------------
+print('Load active analyses')
+
+active_analyses <- readr::read_rds("lib/active_analyses.rds")
+
+table2_names <- gsub(
+  "out_date_",
+  "",
+  unique(
+    active_analyses[
+      active_analyses$cohort ==
+        {
+          cohort
+        },
+    ]$name
+  )
+)
+
+if (focus == "severity") {
```

Comment:
I think we want to generalise this. We almost always just use what is currently called `severity` but for the mental health paper, we were asked to provide this table by history of mental health events (though you have reinterpreted this below as COVID19 history). 

I think we want a parameter for subgroup and that resticts to `main` (which is almost always needed as it is the overall values) and whatever subgroup has been specified (e.g., `sub_covidhospital_`). If a subgroup is not specified for some reason, perhaps we just want it to return the output just for `main`?